packages/amplify-cli-core/API.md

```diff
@@ -938,6 +1053,23 @@ export interface INotificationsResourceMeta {
     service: string;
 }
 
+// @public (undocumented)
+export class Input {
```

Comment:
is this input to cli ?

Reply:
Yes, it's a class that contains information about the original command used to invoke the cli

Reply:
This could use more descriptive name then. `CommandLineInput` ?

Reply:
Done